apollo-client/src/main/java/com/ctrip/framework/apollo/monitor/internal/tracer/ApolloClientMonitorMessageProducer.java

```diff
@@ -124,7 +126,7 @@ private void publishMetaServiceEvent() {
     ApolloClientMonitorEventPublisher.publish(
         ApolloClientMonitorEventFactory.getInstance().createEvent(META_FRESH)
             .withTag(TAG_BOOTSTRAP)
-            .putAttachment(META_FRESH, LocalDate.now().toString()));
+            .putAttachment(META_FRESH, DateUtil.formatLocalDateTime(LocalDateTime.now())));
```

Comment:
Add some unit test to verify it? i.e the NPE

Reply:
done